### PR TITLE
EM-1822 expose api states

### PR DESCRIPTION
**Pull request by tomxelliott**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,22 @@ The toolbar may be toggled off by setting `showToolbase` to false. The toolbar i
...
+      message: error.message
+    }
 
+The list of exceptions thrown by the Media Viewer are as follows:
+- UnknownErrorException
+- MissingPdfException
+- InvalidPDFException
+- UnexpectedResponseException
+- HttpErrorResponse
+- PasswordException
```
Comment (velimd) [inline README.md]: password exception?

File: projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.spec.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { DocumentLoadProgress } from './pdf-js/pdf-js-wrapper';
 import { ViewerEventService } from '../viewer-event.service';
 import { PdfAnnotationService } from './pdf-annotation-service';
 import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
+import {ViewerException} from '../error-message/viewer-exception.model';
```
Comment (velimd) [inline projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.spec.ts]: spacing but not a problem

File: README.md (hunk 1 above)
Comment (tomxelliott) [reply README.md]: These are all possible exceptions that can be thrown from Mozilla Pdf Lib we are using (so including the PasswordException

File: projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.spec.ts (hunk 2 above)
Comment (tomxelliott) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.spec.ts]: Will amend now

**Pull request merged**